<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkPatientTableInspector</class>
 <widget class="QWidget" name="QmitkPatientTableInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Semantic Relations content:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="segmentationNodeButton">
     <property name="text">
      <string>Show segmentation nodes</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">nodeButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="imageNodeButton">
     <property name="text">
      <string>Show image nodes</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">nodeButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="nodeButtonGroup"/>
 </buttongroups>
</ui>
